src/app/admin/approval/project/[id]/page.tsx

```
@@ -248,7 +257,7 @@ export default function ProjectApprovalDetailPage() {
 
             {/* 승인 반려 란 */}
             {project?.isApproved === "PENDING" && (
-            <div className="flex flex-col gap-4">
+            <div className="flex flex-col gap-4" data-cy="project-approve-form">
                 <h2 className="text-2xl font-medium border-b pb-2">프로젝트 승인 여부</h2>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)